docs/http.md

```diff
@@ -437,30 +437,42 @@ step.
   | geojson    | [GeoJSON `LineString`](http://geojson.org/geojson-spec.html#linestring) or [GeoJSON `Point`](http://geojson.org/geojson-spec.html#point) if it is only one coordinate (not wrapped by a GeoJSON feature)|
   
 - `name`: The name of the way along which travel proceeds.
+- `lanes`: the available turn lanes at the turn
```

Comment:
Are these only valid at the turn point or for the whole segment? If it is only valid at the turn this need to be part of `StepManeuver`.